### PR TITLE
#365 Add --ignore-opts-files option

### PR DESCRIPTION
**Pull request by Slakah**
Add a flag `--ignore-opts-files`. This flag will enable the temporary deletion of .sbtopts and .jvmopts files (previously this PR was to have a flag `--disable-ignore-opts-files` to do the opposite).

See: #365 

Please let me know if there's anything amiss or any tests that I should probably add. Thanks!

### CODE REVIEW

File: modules/core/src/main/scala/org/scalasteward/core/application/Cli.scala [hunk 1]
```diff
@@ -42,6 +42,7 @@ object Cli {
       readOnly: List[String] = Nil,
       disableSandbox: Boolean = false,
       doNotFork: Boolean = false,
+      disableIgnoreOptsFiles: Boolean = false,
```
Comment (mwz) [inline modules/core/src/main/scala/org/scalasteward/core/application/Cli.scala]: Perhaps `respectOptsFiles` would be a better name for this flag?

**Comment by fthomas** [comment]
LGTM, thanks @Slakah! I'd be also okay with `--ignore-opts-files` and changing the default behavior when this flag is not present. I agree that removing those files is a special case and warrants an additional flag and the default should be not touching them at all.

**Comment by Slakah** [comment]
> I'd be also okay with --ignore-opts-files and changing the default behavior when this flag is not present.

Done!

I've also added some tests, to check my changes actually work!

**Comment by fthomas** [comment]
@all-contributors please add @Slakah for code, test

**Pull request merged**